tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -1423,6 +1423,7 @@ public function dataFileAsserts(): iterable
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-10302-interface-extends.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-10302-trait-extends.php');
 		yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-10302-trait-implements.php');
+		yield from $this->gatherAssertTypes(__DIR__ . '/data/impure-error-log.php');
```

Comment:
Please don't put at the end, but somewhere random in the middle. This is to prevent conflicts when merging into 1.11.x